### PR TITLE
Fix dynamic_field? method to skip embedded relations from being tracked

### PR DESCRIPTION
**Pull request by JagdeepSingh**

### CODE REVIEW

**Comment by JagdeepSingh** [comment]
@dblock Please review this PR.
Acc to coveralls, a `let` statement in rspecs is missing coverage. Not sure, how to cover that. I have fixed other build failure errors.

@johnnyshields FYI

**Comment by johnnyshields** [comment]
@JagdeepSingh for the let which is missing from coveralls, try doing a non-dynamic class definition. (define the class outside of the let)

**Comment by johnnyshields** [comment]
@dblock please merge this?

**Comment by dblock** [comment]
It's missing a CHANGELOG entry please.

**Comment by johnnyshields** [comment]
@JagdeepSingh please add a changelog entry here?

**Comment by JagdeepSingh** [comment]
@dblock Added the CHANGELOG entry.

**Comment by johnnyshields** [comment]
@dblock I'm merging this as changelog is done.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 ------------
 
 * Your contribution here.
+* [#162](https://github.com/aq1018/mongoid-history/pull/162): Do not consider embedded relations as dynamic fields - [@JagdeepSingh](https://github.com/JagdeepSingh)
```
Comment (dblock) [inline CHANGELOG.md]: Missing a period at the end ;)

**Comment by dblock** [comment]
Looking good, thanks @johnnyshields.

**Pull request merged**